### ISSUE TITLE
pulumi import panic: runtime error

### ISSUE THREAD

**Issue by huydinhle** [pulumi/pulumi#5883]
pulumi import panic with nil pointer 
## Expected Behavior

The command should be executed successfully
## Current Behavior

The command returns panic instead
```❯ pulumi import gcp:pubsub/topic:opic trigger-heartbeat trigger-heartbeat
warning: Pulumi will not be able to print a statefile permalink using these credentials. Neither a GoogleAccessID or PrivateKey are available. Try using a GCP Service Account.
Enter your passphrase to unlock config/secrets
    (set PULUMI_CONFIG_PASSPHRASE or PULUMI_CONFIG_PASSPHRASE_FILE to remember):
Previewing import (dev):
     Type                 Name                     Plan
     pulumi:pulumi:Stack  heartbeat-scheduler-dev  running
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x4a81a96]

goroutine 103 [running]:
github.com/pulumi/pulumi/pkg/v2/resource/deploy.(*deploymentExecutor).importResources(0xc00066a620, 0x5c10ac0, 0xc000672f30, 0x10a844a8, 0xc000cf7540, 0x7fffffff, 0x0, 0x0, 0x0, 0x0, ...)
        /private/tmp/pulumi-20201206-63899-1azabtx/pkg/resource/deploy/deployment_executor.go:459 +0x276
github.com/pulumi/pulumi/pkg/v2/resource/deploy.(*deploymentExecutor).Execute(0xc00066a620, 0x5c10ac0, 0xc000672f30, 0x10a844a8, 0xc000cf7540, 0x7fffffff, 0x0, 0x0, 0x0, 0x0, ...)
        /private/tmp/pulumi-20201206-63899-1azabtx/pkg/resource/deploy/deployment_executor.go:138 +0x1247
github.com/pulumi/pulumi/pkg/v2/resource/deploy.(*Deployment).Execute(...)
        /private/tmp/pulumi-20201206-63899-1azabtx/pkg/resource/deploy/deployment.go:383
github.com/pulumi/pulumi/pkg/v2/engine.(*deployment).run.func1(0x5c1b700, 0xc000cf7540, 0xc0014b4280, 0x5c10ac0, 0xc000672f30, 0xc000672f01, 0xc0014d7700, 0xc0006768a0)
        /private/tmp/pulumi-20201206-63899-1azabtx/pkg/engine/deployment.go:251 +0x267
created by github.com/pulumi/pulumi/pkg/v2/engine.(*deployment).run
        /private/tmp/pulumi-20201206-63899-1azabtx/pkg/engine/deployment.go:237 +0x306
```
## Steps to Reproduce

Run a simple `pulumi import` command with gcp as the provider, and python is the runtime
```
❯ pulumi import gcp:pubsub/topic:opic trigger-heartbeat trigger-heartbeat
```

**Comment by stack72** [comment]
Hi @huydinhle 

My apologies you are seeing this issue - I just opened a PR that fixes the problem and it will be made available in Pulumi v2.15.4 later today

Paul

**Comment by huydinhle** [comment]
Thank you
